pages.la/common/tldr.md

```diff
@@ -0,0 +1,20 @@
+# tldr
+
+> ສະແດງຕົວຢ່າງແບບງ່າຍສຳລັບເຄື່ອງມືເທິງ command-line ຈາກໂປຣເຈັກ tldr-pages.
+> ຂໍ້ມູນເພີ່ມເຕີມ: <https://tldr.sh>.
+
+- ສະແດງຕົວຢ່າງການໃຊ້ຄຳສັ່ງທີ່ໃຊ້ເລື້ອຍໆ (ຄຳໃບ້: ນີ້ເປັນເຫດຜົນທີ່ທ່ານໃຊ້ບໍລິການຂອງເຮົາແມ່ນບໍ່ລ່ະ!):
+
+`tldr {{command}}`
+
+- ສະແດງໜ້າ tldr ຂອງຄຳສັ່ງ tar ສຳລັບລະບົບປະຕິບັດການ Linux:
```

Comment:
Change this command to better align with [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldr.md).